packaging/files/usr/lib/systemd/system/supervisord.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=Supervisor daemon
+Documentation=http://supervisord.org
+After=network.target
+
+[Service]
+ExecStart=/usr/bin/supervisord -n -c /etc/supervisord.conf
+ExecStop=/usr/bin/supervisorctl -c /etc/supervisord.conf shutdown
+ExecReload=/usr/bin/supervisorctl -c /etc/supervisord.conf reload
+KillMode=process
+Restart=on-failure
+RestartSec=42s
```

Comment:
42?

Reply:
This should be changed as I was testing, will update that one around. 10 or something

Reply:
Updated to 10s